wordpress-mobile/WordPress-iOS
# Implement action to redirect the user to Privacy Settings screen

Fixes #20799

## Description

This PR provides an API to present the Privacy Settings screen. The API can be used like this:

```swift
RootViewCoordinator.sharedPresenter.navigateToPrivacySettings()
```

https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/54a8f5dd-6de5-465c-93d9-dc7e98b5fca6

## Test Instructions

**⚠️ Please perform the following test case on both an iPhone and iPad.**

1. Add the following code in `BlogDashboardViewController.viewDidAppear`
```swift
RootViewCoordinator.sharedPresenter.navigateToPrivacySettings()
```
2. Build and run the app.
3. **Expect** the app to navigate to the Privacy Settings screen passing by the "Me" and "App Settings" screens.

## Regression Notes
1. Potential unintended areas of impact
The "Privacy Settings" navigation logic from the "App Settings" screen was slightly refactored. Navigating to "Privacy Settings" screen manually should be tested.

    1. Tap "Me" button.
    2. Tap "App Settings".
    3. Tap "Privacy Settings".

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

3. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
